portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tal's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <ul>
+        <li><a class="active" href="index.html">Home</a></li>
+        <li><a href="hobbies.html">Hobbies</a></li>
+        <li><a href="dartmouth.html">School/Profession</a></li>
+        <li><a href="blog.html">Blog</a></li>
+
+    </ul>
+    <p> </p>
+
+   
+    <div id="content">
+      <h1>My Hobbies</h1>
+      <img class = 'img' src="/images/surf2.png" width = "30%" height = "30%"/>
+
+      <p>Surfing: Despite the fact I live about 45 minutes away from any surfing spots, 
+          I still like to surf every weekend. My favorite spots are Nantasket Beach, 
+          Rye Rocks, and the Wall. I've surfed along the California Coast
+          and in the Dominican Republic! 
+      </p>
+
+      <p>Hiking: I like to spend a lot of time outside. Luckily, I go to school 
+          in the mountains and I'm able to get out a lot. In the winter, I also like
+          to go skiing!
+      </p>
+
+      <p>If you're a surfer too, check out Magic Seaweed! It is a 
+          website/app that allows you to access a ton of information
+          about different surf spots around the world. The information
+          ranges from current surf conditions to local tips and warnings</p>
+    
+      <button class = 'MSW' onclick="window.location.href='https://magicseaweed.com/';">
```

Comment:
button works here, but <a href=""> would be the recommended approach.  [There's this suggestion](https://google.github.io/styleguide/htmlcssguide.html#Semantics) for using the semantics the HTML tag was intended for.  Specially buttons would usually be for triggering behavior on the same page, while anchor tag would be for navigating to a different page.